api/app/src/routes/medical/patient.ts

```diff
@@ -162,14 +162,14 @@ router.delete(
  * Gets all patients corresponding to the specified facility at the customer's organization.
  *
  * @param   req.cxId              The customer ID.
- * @param   req.query.facilityId  The ID of the facility the user patient is associated with.
+ * @param   req.query.facilityId  The ID of the facility the user patient is associated with (optional).
  * @return  The customer's patients associated with the given facility.
  */
 router.get(
   "/",
   asyncHandler(async (req: Request, res: Response) => {
     const cxId = getCxIdOrFail(req);
-    const facilityId = getFromQueryOrFail("facilityId", req);
+    const facilityId = getFrom("query").optional("facilityId", req);
```

Comment:
Allows to query all patients of a cx - since its local DB only thought it could be useful, but can remove if preferred (I used for some tests local).

Reply:
Makes sense but should we consider another route for this?

Reply:
Could add a /facility/:id/patients in the facilities route section and then have this be all patients

Reply:
I think it makes sense as is, you hit `GET /patient` to get Patients... if you add a `facilityId` param it will filter by that.

Don't you think so?

Reply:
That makes sense as well, just considering maybe a better separation of concerns but cool to merge as is